integration-tests/src/test/java/oracle/kubernetes/operator/BaseTest.java

```diff
@@ -168,12 +168,53 @@ public void testAdminT3Channel(Domain domain) throws Exception {
     Boolean exposeAdmint3Channel = (Boolean) domainMap.get("exposeAdminT3Channel");
 
     if (exposeAdmint3Channel != null && exposeAdmint3Channel.booleanValue()) {
+      ExecResult result =
+          TestUtils.kubectlexecNoCheck(
+              domain.getDomainUid() + ("-") + domainMap.get("adminServerName"),
+              "" + domainMap.get("namespace"),
+              " -- mkdir -p " + appLocationInPod);
+      if (result.exitValue() != 0) {
+        throw new RuntimeException(
+            "FAILURE: command to create directory "
+                + appLocationInPod
+                + " in the pod failed, returned "
+                + result.stderr()
+                + " "
+                + result.stdout());
+      }
+
       domain.deployWebAppViaWLST(
           TESTWEBAPP,
           getProjectRoot() + "/src/integration-tests/apps/testwebapp.war",
+          appLocationInPod,
           getUsername(),
           getPassword());
       domain.verifyWebAppLoadBalancing(TESTWEBAPP);
+
+      // this check is done for domain-home-in-image, it needs 12.2.1.3 patched image
+      // otherwise managed servers will be unicast errors after app deployment
+      if (domainMap.containsKey("domainHomeImageBase")) {
+        if (domainMap.get("initialManagedServerReplicas") != null
+            && ((Integer) domainMap.get("initialManagedServerReplicas")).intValue() >= 1) {
+
+          result =
+              ExecCommand.exec(
+                  "kubectl logs "
+                      + domain.getDomainUid()
+                      + ("-")
+                      + domainMap.get("managedServerNameBase")
+                      + "1 -n "
+                      + domainMap.get("namespace")
+                      + " | grep BEA-000802");
```

Comment:
What is the contents of BEA-000802?  I'd like to understand how this ensures teh patch is applied...

Reply:
When domain home on image test was using a non-patched 12.2.1.3 image, seeing unicast errors in managed server logs which makes them run as standalone servers, not part of the cluster. The servers would still be running, so had to check for the error message. To get the full error message, I need to look into the logs by running the test on a non-patched 12.2.1.3 image.

Reply:
OK - that helps.  Since this is pretty obscure, but helpful to understand, I think it'd be nice to include your comment as a code comment.

Reply:
Sure. here is the error 
<Jan 18, 2019 8:54:16,214 PM GMT> <Error> <Kernel> <BEA-000802> <ExecuteRequest failed
 java.lang.AssertionError: LocalGroup should atleast have the local server!.
java.lang.AssertionError: LocalGroup should atleast have the local server!
	at weblogic.cluster.messaging.internal.GroupImpl.send(GroupImpl.java:176)
	at weblogic.cluster.messaging.internal.server.UnicastFragmentSocket.send(UnicastFragmentSocket.java:97)
	at weblogic.cluster.FragmentSocketWrapper.send(FragmentSocketWrapper.java:84)
	at weblogic.cluster.UnicastSender.send(UnicastSender.java:53)
	at weblogic.cluster.UnicastSender.send(UnicastSender.java:21)
	Truncated. see log file for complete stacktrace

Reply:
It'd be helpful to cut-and paste the log message and the stack trace into the code as a comment...

Reply:
Ok

Reply:
Looks good! Thanks.